integ-test/build.gradle

```diff
@@ -38,7 +38,7 @@ apply plugin: 'java'
 apply plugin: 'io.freefair.lombok'
 apply plugin: 'com.wiredforcode.spawn'
 
-String baseVersion = "2.8.0"
+String baseVersion = "2.9.0"
```

Comment:
To save us from doing this version update every time, is there another string in the project this file can reference to have the correct version?

Reply:
No :(